sdk/storage/platform-matrix.json

```diff
@@ -14,21 +14,21 @@
     },
     {
       "Agent": { "macos-11": { "OSVmImage": "macos-11", "Pool": "Azure Pipelines" } },
-      "JavaTestVersion": "1.11",
+      "JavaTestVersion": "1.21",
```

Comment:
I would revert this back to Java 11

Reply:
I see no issue with that, I just wonder why. Is it to keep things stable?

Reply:
I don't have a strong reason, but I also don't know why this was using Java 11 and whether that was important. Should follow-up with Storage on this